types/react-native-elements/index.d.ts

```diff
@@ -250,4 +400,1126 @@ export interface BadgeProperties {
+    featuredSubtitleStyle?: TextStyle;
+
+    /**
+     * Additional divider styling
+     * (if title provided)
+     */
+    dividerStyle?: ViewStyle;
+
+    /**
+     * Specify different font family
+     *
+     * @default System font bold (iOS), Sans Serif Bold (android)
+     */
+    fontFamily?: string;
+
+    /**
+     * Specify image styling if image is provided
+     */
+    imageStyle?: ImageStyle;
+
+    /**
+     * Specify styling for view surrounding image
+     */
+    imageWrapperStyle?: ViewStyle;
+
+    /**
+     * Add an image as the heading with the image prop
+     */
+    image?: ImageURISource;
+}
+
+/**
+ * Card component
+ *
+ * @see https://react-native-training.github.io/react-native-elements/API/card/
+ */
+export class Card extends React.Component<CardProps, any> {}
+
+/**
+ * Set the buttons within a Group.
+ */
+export interface ElementObject {
+    element: JSX.Element | React.ReactType;
+}
+
+/**
+ * Set the border styles for a component.
+ *
+ * @export
+ * @interface InnerBorderStyleProperty
+ */
+export interface InnerBorderStyleProperty {
+    color?: string;
+    width?: number;
+}
+
+export interface ButtonGroupProps {
+    /**
+     * Current selected index of array of buttons
+     */
+    selectedIndex: number;
+
+    /**
+     * Method to update Button Group Index
+     */
+    // onPress(): void;
+    onPress(selectedIndex: number): void;
+
+    /**
+     * Array of buttons for component, if returning a component, must be an object with { element: componentName }
+     */
+    buttons: string[] | ElementObject[];
+
+    /**
+     * Choose other button component such as TouchableOpacity
+     *
+     * @default TouchableHighlight
+     */
+    component?: React.ComponentClass;
+
+    /**
+     * Specify styling for main button container
+     */
+    containerStyle?: ViewStyle;
+
+    /**
+     * inherited styling	object (style)	specify styling for button
+     */
+    buttonStyle?: ViewStyle;
+
+    /**
+     * Specify color for selected state of button
+     *
+     * @default 'white'
+     */
+    selectedBackgroundColor?: string;
+
+    /**
+     * Specify specific styling for text
+     */
+    textStyle?: TextStyle;
+
+    /**
+     * Specify specific styling for text in the selected state
+     */
+    selectedTextStyle?: TextStyle;
+
+    /**
+     * inherited styling	object { width, color }	update the styling of the interior border of the list of buttons
+     */
+    innerBorderStyle?: InnerBorderStyleProperty;
+
+    /**
+     * Specify underlayColor for TouchableHighlight
+     *
+     * @default 'white'
+     */
+    underlayColor?: string;
+
+    /**
+     * Disables the currently selected button if true
+     *
+     * @default false
+     */
+    disableSelected?: boolean;
+
+    /**
+     * Determines what the opacity of the wrapped view should be when touch is active.
+     */
+    activeOpacity?: number;
+
+    /**
+     * Border radius for the container
+     *
+     * @type {number}
```

Comment:
Comments like these you can remove "type, memberof, export" they just add bloat.